flink-end-to-end-tests/test-scripts/common_mesos_docker.sh

```diff
@@ -52,11 +54,19 @@ function start_flink_cluster_with_mesos() {
     set_config_key "jobmanager.rpc.address" "mesos-master"
     set_config_key "rest.address" "mesos-master"
 
-    docker exec -itd mesos-master bash -c "${FLINK_DIR}/bin/mesos-appmaster.sh -Dmesos.master=mesos-master:5050"
+    docker cp ${FLINK_DIR} mesos-master:$MESOS_FLINK_DIR
+    docker cp ${END_TO_END_DIR}/test-scripts mesos-master:$MESOS_END_TO_END_DIR
```

Comment:
Why do we need to copy the `MESOS_END_TO_END_DIR` directory to docker container? If you want to use the data, just copy them.